src/components/overlays/bottom-sheet/RuiBottomSheet.stories.ts

```diff
@@ -0,0 +1,93 @@
+import RuiButton from '@/components/buttons/button/RuiButton.vue';
+import RuiCard from '@/components/cards/RuiCard.vue';
+import RuiBottomSheet, { type BottomSheetProps } from './RuiBottomSheet.vue';
```

Comment:
Can we do a full import from `@/components/` as with the button/card for consistency?